README.md

```diff
@@ -1,13 +1,53 @@
 ### Gentoo Updater
 
-The project aims to develop a comprehensive tool to completely 
-automating Gentoo Linux upgrades. This utility will:
-* do full system upgrade (update `@world`) or only install security updates
-* parse output from `Portage` and compose an update report
-* send the upgrade report via email, IRC or a mobile app
-
+`gentoo-update` is a tool that automates installing updates on Gentoo Linux. 
+By default it only installs security updates from [GLSA](https://security.gentoo.org/glsa/), 
+but it also provides support to update `@world`.  
 
 This project 
 [originates](https://wiki.gentoo.org/wiki/Google_Summer_of_Code/2023/Ideas/Automated_Gentoo_system_updater) 
 from 2023 Google Summer of Code.
 
+
+#### Usage
+`gentoo-update` can be easily installed with pip (ebuild coming soon):
+```bash
+pip install gentoo_update
+```
+
+Here are some usage examples:
+* Basic security update
+```bash
+gentoo-update
+```
+
+* Full system update with extra update parameters
+```bash
+gentoo-update --update-mode full --args color=n
+```
+
+* Full system update, show elogs and news
+```bash
+gentoo-update --update-mode full --read-logs y --read-news y
+```
+
+The detailed explanation of command flags can be found in `--help`.  
+
+
+#### Testing
+**Note** Testing will be automated in the future, right now it's a bit hackish
+In `gentoo_update/tests` there is a Docker Compose file that 
+can be used for testing. It builds containers based on stage3 
+tarballs and mounts the source code to `/root/gentoo_update_source`.  
+
+Example of a test:
```

Comment:
Add how long the tests are expected to take

Reply:
Unit tests are instant (0.002s), but they are very basic at this stage.

But the tests in Docker container take a while.
On my system, 8GB RAM 6CPUs and 500kbp/s bandwith, it takes about 4 minutes to build a systemd base stage3 image, and an additional 5-10 mins to run updates/tests. This varies because installed packages are different every time.
If it's the desktop image then it's safe to multiply all time on *2, so about 20-25 minutes.